packages/react/src/auto/polaris/PolarisAutoTable.tsx

```diff
@@ -168,22 +169,24 @@ const PolarisAutoTableComponent = <
         ids={selection.recordIds}
         selectedRows={selectedRows}
       />
-      <IndexFilters
-        mode={mode}
-        setMode={setMode}
-        appliedFilters={[]}
-        filters={[]}
-        onClearAll={() => undefined}
-        tabs={[]}
-        selected={1}
-        loading={fetching}
-        cancelAction={{ onAction: () => search.clear() }}
-        disabled={!!error}
-        // Search
-        queryValue={search.value}
-        onQueryChange={search.set}
-        onQueryClear={search.clear}
-      />
+      {!hideSearching && (
```

Comment:
Right now we don't have any other filter options other than the full-text search, so we can just hide the whole thing.